setup.py

```diff
@@ -7,33 +7,21 @@
 
 project_base_url = 'https://github.com/LostFan123/pode/'
 
-install_requires = [
-    'lz>=0.8.1',
-    'matplotlib>=2.2.2',
-    'networkx>=2.4',
-    'numpy>=1.16.3',
-    'shapely==1.6.4.post2'
-]
-setup_requires = [
-    'pytest-runner>=4.2',
-]
-tests_require = [
-    'pytest>=3.8.1',
-    'pytest-cov>=2.6.0',
-    'hypothesis>=3.73.1',
-]
-
-setup(name='pode',
+setup(name=pode.__name__,
       packages=find_packages(exclude=('tests', 'tests.*')),
       version=pode.__version__,
       description=pode.__doc__,
       long_description=Path('README.md').read_text(encoding='utf-8'),
       long_description_content_type='text/markdown',
       author='Georgy Skorobogatov',
-      author_email='skorobogatov@phystech.edu',
+      author_email='georgy.skorobogatov@upc.edu',
+      classifiers=[
+          'License :: OSI Approved :: MIT License',
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: Implementation :: CPython',
```

Comment:
why not `PyPy` as well? it looks like it can work in it

Reply:
I wanted to make sure first that it would run on any version at all, and later extend it to other ones.